kaznet/apps/ona/tasks.py

```diff
@@ -220,3 +222,36 @@ def task_sync_xform_can_submit_checks():
     xforms = XForm.objects.filter(deleted_at=None)
     for xform in xforms:
         task_check_if_users_can_submit_to_form.delay(xform_id=xform.id)
+
+
+# pylint: disable=not-callable
+@celery_task(name="task_sync_submission_review")
+def task_sync_submission_review(instance_id, kaznet_review_status, comment):
+    """
+    Sync auto review of submission with its review on onadata
+    """
+    ona_review_status = convert_kaznet_to_ona_submission_status(
+        kaznet_review_status)
+    sync_submission_review(instance_id, ona_review_status, comment)
+
+# pylint: disable=not-callable
+@celery_task(name="task_sync_outdated_submission_review")
+def task_sync_outdated_submission_reviews():
+    """
+    Sync outdated submission reviews that did not
+    sync with onadata when they were created
+    """
+    # query all instances from db and iterate through,
+    # calling sync_submission_review_for_each
+    all_instances = Instance.objects.all()
```

Comment:
Instead of getting all instances can we filter to only get the ones that do not have `synced_with_ona_data` as True?  If we do this we shall also avoid the necessity of the if block in line 249 below.